src/index.html

```diff
@@ -11,8 +11,68 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <article class="card__element">
+        <div class="card__bc-image"></div>
+      </article>
+
+      <article class="card__element">
+        <div class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+      </article>
+
+      <article class="card__element">
+        <div class="card__product-code">Product code: 195434</div>
+      </article>
```

Comment:
not the best practice on using `article` and `div` inside of it

[read more about better usage of semantic tags](https://www.w3schools.com/html/html5_semantic_elements.asp)